### DISCUSSION TITLE
Default Username and Password

### DISCUSSION THREAD

**Discussion by BitterMonkey** [inventree/InvenTree#2949] [General]
I have just set up Inventree on a Pi4 as bare metal setup and have the server running, however im not sure what the default user name as password would be set up as. I don't remember setting up a username or password as i was setting it up. The server is running and i am trying to log in via a browser over the network.

Edit - Never mind it is solved. I reread the instructions and figured out the superuser credentials

**Comment by SchrodingersGat** [comment]
You need to configure an administrator user account as part of the setup process:

https://inventree.readthedocs.io/en/latest/start/install/#create-admin-account

**Reply by BitterMonkey** [reply]
Thanks i Re read the instructions and found that and im in now.
